tests/system/action/committee/test_json_upload.py

```diff
@@ -0,0 +1,298 @@
+                    }
+                ]
+            },
+        )
+        self.assert_status_code(response, 200)
+        assert response.json["results"][0][0]["rows"][0] == {
+            "state": ImportState.NEW,
+            "messages": [],
+            "data": {
+                "name": "test",
+                "meeting_name": "test meeting",
+                "meeting_template": {"value": "test", "info": ImportState.WARNING},
+            },
+        }
+
+    def test_json_upload_meeting_template_found(self) -> None:
+        self.set_models({"meeting/23": {"name": "test"}})
+        response = self.request(
+            "committee.json_upload",
+            {
+                "data": [
+                    {
+                        "name": "test",
+                        "meeting_name": "test meeting",
+                        "meeting_template": "test",
+                    }
+                ]
+            },
+        )
+        self.assert_status_code(response, 200)
+        assert response.json["results"][0][0]["rows"][0] == {
+            "state": ImportState.NEW,
+            "messages": [],
+            "data": {
+                "name": "test",
+                "meeting_name": "test meeting",
+                "meeting_template": {
+                    "value": "test",
+                    "info": ImportState.DONE,
+                    "id": 23,
+                },
+            },
+        }
+        assert response.json["results"][0][0]["statistics"][5] == {
+            "name": "Meetings copied from template",
+            "value": 1,
+        }
+        assert response.json["results"][0][0]["statistics"][4] == {
+            "name": "Meetings created without template",
+            "value": 0,
+        }
+
+    def test_json_upload_committee_managers(self) -> None:
+        self.set_models({"user/23": {"username": "test"}})
+        response = self.request(
+            "committee.json_upload",
+            {
+                "data": [
+                    {
+                        "name": "committee A",
+                        "committee_managers": '"test", "new"',
+                    }
+                ]
+            },
+        )
+        self.assert_status_code(response, 200)
+        assert response.json["results"][0][0]["rows"][0] == {
+            "state": ImportState.NEW,
+            "messages": [],
+            "data": {
+                "name": "committee A",
+                "committee_managers": [
+                    {"value": "test", "info": ImportState.DONE, "id": 23},
+                    {"value": "new", "info": ImportState.WARNING},
+                ],
+            },
+        }
+        assert response.json["results"][0][0]["statistics"][6] == {
+            "name": "Committee managers relations",
+            "value": 2,
+        }
+
+    def test_json_upload_committee_managers_wrong_json(self) -> None:
+        self.set_models({"user/23": {"username": "test"}})
+        response = self.request(
```

Comment:
This error should not be possible, because in JSON the string[] is a comma separated list and we will convert it to string[] by putting surrounding brackets.